pip/commands/list.py

```diff
@@ -53,7 +58,7 @@ def __init__(self, *args, **kw):
             help=('If in a virtualenv that has global access, do not list '
                   'globally-installed packages.'),
         )
-        self.cmd_opts.add_option(
+        cmd_opts.add_option(
             '--user',
```

Comment:
This looks like an unrelated bug-fix? If so, could you split it out into a separate PR?